### ISSUE TITLE
About ogbg-molhiv results 

### ISSUE THREAD

**Issue by dongZheX** [lightaime/deep_gcns_torch#87]
Hello! (*^_^*)， thanks for the code.
I used the following command to train on the molhiv dataset：
`python main.py --use_gpu --conv_encode_edge --num_layers 7 --dataset ogbg-molhiv --block res+ --gcn_aggr softmax --t 1.0 --learn_t --dropout 0.2`
Why I can't reproduce the results in [LeaderBoard？Is](https://ogb.stanford.edu/docs/leader_graphprop/#ogbg-molhiv) it something wrong with me?

**My result is :**
VAL_ROCAUC | TEST_ROCAUC
0.7976±0.0179 | 0.7744 ± 0.1234

**LeaderBoard result:**
VAL_ROCAUC | TEST_ROCAUC
0.8427 ± 0.0063 | 0.7858 ± 0.0117

**Comment by lightaime** [comment]
Hi @dongZheX, could you please check again your results or provide the logs files you have. Our results on the ogb leaderboard is: Val: 0.8427 ± 0.0063, and Test:  0.7858 ± 0.0117. The val ROCAUC you got seems not reasonable.

**Comment by dongZheX** [comment]
> Hi @dongZheX, could you please check again your results or provide the logs files you have. Our results on the ogb leaderboard is: Val: 0.8427 ± 0.0063, and Test: 0.7858 ± 0.0117. The val ROCAUC you got seems not reasonable.

It's my fault.  I use a new method to record the results but have something wrong in it.
